Frameworks/BlockchainCertificates/Issuer.swift

```diff
@@ -130,7 +145,8 @@ public struct Issuer {
     /// This is the inverse of `toDictionary`
     ///
     /// - parameter dictionary: A set of key-value pairs with data used to create the Issuer object
-    public init(dictionary: [String: Any]) throws {
+    /// - parameter version: Version hint for parsing
+    public init(dictionary: [String: Any], asVersion version: IssuerVersion = IssuerVersion.one) throws {
```

Comment:
Just a Swift tip here. Since you've specified the type of the enum, you can leave off the enum type in the assignment

```
asVersion version: IssuerVersion = .one) throws {
```